vignettes/age-strat-rates.Rmd

```diff
@@ -0,0 +1,252 @@
+---
+title: "Age-stratified hospitalisation and death rates"
+output: 
+  bookdown::html_vignette2:
+    code_folding: show
+pkgdown:
+  as_is: true
+vignette: >
+  %\VignetteIndexEntry{Age-stratified hospitalisation and death rates}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+::: {.alert .alert-info}
+If you are unfamiliar with the {simulist} package or the `sim_linelist()` function [Get Started vignette](simulist.html) is a great place to start.
+:::
+
+This vignette will describe how to simulate a line list with age-stratified (or age-dependent) hospitalisation and death rates. 
```

Comment:
Worth making sure we're using rates and risks consistently throughout. Although literature often uses 'case fatality rate' or 'case hospitalisation rates', these are actually risks. So if the function involves the probability of an event i.e. P(hospitalisation | case, age), we should use risk I think.

Reply:
I agree, I will make this change in a dedicated PR.